components/movementsensor/gpsrtk/rtk_station.go

```diff
@@ -365,7 +365,7 @@ func (r *rtkStation) CompassHeading(ctx context.Context) (float64, error) {
 	return 0, movementsensor.ErrMethodUnimplementedCompassHeading
 }
 
-func (r *rtkStation) Readings(ctx context.Context) (map[string]interface{}, error) {
+func (r *rtkStation) Readings(ctx context.Context, _ map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
(q) why the wildcard here when we've used the `extra` name in other instances?

Reply:
honestly, I happened to notice that this one wasn't actually using the argument but was flying through the rest. I can change to `extra` just to be consistent.

Reply:
lol totally fair. I think most other features have used `extra` pretty consistently so I mildly prefer that, but honestly it's not a big deal either way!